cpp/src/randomforest/randomforest.hpp

```diff
@@ -76,7 +76,7 @@ struct RF_params {
 
 void set_rf_params(RF_params& params, int cfg_n_trees = 1,
                    bool cfg_bootstrap = true, float cfg_rows_sample = 1.0f,
-                   int cfg_n_streams = 4);
+                   int cfg_n_streams = 8);
```

Comment:
So 8 is the optimal number after the changes in this PR?

Reply:
8 should work better. 4 was a bit conservative.